service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -102,58 +92,76 @@ public void notifyAdmin(final AdditionalApplicationsUploadedEvent event) {
         }
     }
 
-    @SuppressWarnings("unchecked")
-    private void sendNotificationToLocalAuthorityAndDigitalRepresentatives(final CaseData caseData,
-                                                                           List<Element<Other>> othersSelected,
-                                                                           NotifyData notifyData) {
-        Set<String> digitalRepresentatives = representativesInbox.getEmailsByPreference(caseData, DIGITAL_SERVICE);
-        Set<String> digitalRecipientsOtherNotNotified = (Set<String>) otherRecipientsInbox.getNonSelectedRecipients(
-            DIGITAL_SERVICE, caseData, othersSelected, element -> element.getValue().getEmail()
-        );
-        digitalRepresentatives.removeAll(digitalRecipientsOtherNotNotified);
-
-        sendToLocalAuthority(caseData, notifyData);
-
-        representativeNotificationService.sendNotificationToRepresentatives(
-            caseData.getId(),
-            notifyData,
-            digitalRepresentatives,
-            INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE_PARTIES_AND_OTHERS
-        );
-    }
+    @EventListener
+    @Async
+    public void notifyLocalAuthority(final AdditionalApplicationsUploadedEvent event) {
+        if (featureToggleService.isServeOrdersAndDocsToOthersEnabled()) {
```

Comment:
if you are interested you can remove the feature toggle service bean from this class and instead have any event listeners annotated like so
```
    @EventListener(condition = "@featureToggleService.isServeOrdersAndDocsToOthersEnabled()")
```
I think this would be much more complex to test though so I am 100% happy keeping as it is, just thought it was neat.